lib/livebook_web/live/session_live.ex

```diff
@@ -2169,7 +2169,9 @@ defmodule LivebookWeb.SessionLive do
       section_views: section_views(data.notebook.sections, data),
       bin_entries: data.bin_entries,
       secrets: data.secrets,
-      hub: Livebook.Hubs.fetch_hub!(data.notebook.hub_id),
+      hub:
+        Livebook.Hubs.get_offline_hub(data.notebook.hub_id) ||
+          Livebook.Hubs.fetch_hub!(data.notebook.hub_id),
```

Comment:
`notebook.hub_id` is never offline hub

```suggestion
      hub: Livebook.Hubs.fetch_hub!(data.notebook.hub_id),
```